Automatic Validation ended with: 
> 2026-03-20T00:50:01.3640413Z ##[error] Manifest Error: The manifest version in the schema header does not match the ManifestVersion property value in the manifest. Value: 1.10.0 Line: 1, Column: 78 File: 6f519626fb884a4536b7005da36554d05bf546290f3b436379cc8c5c2d85443a Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.defaultLocale.1.10.0.schema.json Line: 1, Column: 25 File: 6f519626fb884a4536b7005da36554d05bf546290f3b436379cc8c5c2d85443a Manifest Error: The manifest version in the schema header does not match the ManifestVersion property value in the manifest. Value: 1.10.0 Line: 1, Column: 72 File: a0e7b9c4984c83598d0b96eac3de6b8fac0f0c5fcb0c89d3051522320535181e Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.version.1.10.0.schema.json Line: 1, Column: 25 File: a0e7b9c4984c83598d0b96eac3de6b8fac0f0c5fcb0c89d3051522320535181e Manifest Error: The manifest version in the schema header does not match the ManifestVersion property value in the manifest. Value: 1.10.0 Line: 1, Column: 74 File: e2a8af07d5bd699cf3082eac20f506df91fc2066be552934669de4c6a4c37b9b Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.installer.1.10.0.schema.json Line: 1, Column: 25 File: e2a8af07d5bd699cf3082eac20f506df91fc2066be552934669de4c6a4c37b9b

(Automated response - build 1237.)

--- a/manifests/w/WestWind/DocumentationMonster/0.5.0.0/WestWind.DocumentationMonster.installer.yaml
+++ b/manifests/w/WestWind/DocumentationMonster/0.5.0.0/WestWind.DocumentationMonster.installer.yaml
@@ -1,4 +1,4 @@
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.12.0.schema.json
 
 PackageIdentifier: WestWind.DocumentationMonster
 PackageVersion: 0.5.0.0

--- a/manifests/w/WestWind/DocumentationMonster/0.5.0.0/WestWind.DocumentationMonster.locale.en-US.yaml
+++ b/manifests/w/WestWind/DocumentationMonster/0.5.0.0/WestWind.DocumentationMonster.locale.en-US.yaml
@@ -1,4 +1,4 @@
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.12.0.schema.json
 
 PackageIdentifier: WestWind.DocumentationMonster
 PackageVersion: 0.5.0.0

--- a/manifests/w/WestWind/DocumentationMonster/0.5.0.0/WestWind.DocumentationMonster.yaml
+++ b/manifests/w/WestWind/DocumentationMonster/0.5.0.0/WestWind.DocumentationMonster.yaml
@@ -1,4 +1,4 @@
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.10.0.schema.json
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.12.0.schema.json
 
 PackageIdentifier: WestWind.DocumentationMonster
 PackageVersion: 0.5.0.0

